keyboards/wolfmarkclub/wm1/rules.mk

```
@@ -0,0 +1,37 @@
+# Build Options
+#   change yes to no to disable
+#
+
+# MCU name
+MCU = STM32F103
+
+# GENERIC STM32F103C8T6 board - stm32duino bootloader
+OPT_DEFS = -DCORTEX_VTOR_INIT=0x10000
+MCU_LDSCRIPT = wm1_f103
+BOARD = STM32_F103_STM32DUINO
+
+DFU_ARGS = -d 1eaf:0003 -a2 -R
+DFU_SUFFIX_ARGS = -v 1eaf -p 0003
+
+
```

Stop `:flash` from doing stuff
```suggestion

PROGRAM_CMD=echo 'CLI flashing not supported' >&2

```